mediorum/server/transcode.go

```diff
@@ -364,6 +367,7 @@ func (ss *MediorumServer) transcodeFullAudio(upload *Upload, temp *os.File, logg
 		return onError(err, upload.Status, "os.Open")
 	}
 	defer dest.Close()
+	defer os.Remove(dest.Name())
```

Comment:
probably just an edge case, but maybe this line and the `dest.Close() `should be before running ffmpeg? it's been mostly working fine as is tho